lib/hal/video.c

```diff
@@ -201,7 +201,13 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	
 	DWORD dwAdapter = dwEnc & 0x000000FF;
 	DWORD dwStandard = dwEnc & 0x0000FF00;
-	
+
+	bool is_hd;
```

Comment:
Declare where you set it, not up here (which is also in the middle of the function)

Reply:
Fixed. :+1: